bycycle/burst/utils.py

```diff
@@ -108,6 +109,28 @@ def recompute_edges(df_features, threshold_kwargs):
                             enumerate(burst_edges)])
 
     # Recompute is_burst
+    for idx, cyc_idx in enumerate(burst_edges):
+
+        if idx % 2 == 0:
```

Comment:
this is a little opaque to me. Why do we do "next" if this index is even? Is it because that would indicate the start of a burst? If so, I think the code could be written in a way that this is more obvious.

Like instead of `burst_edges` containing both the starting and stopping edges of a burst, maybe it'd be better to refactor this to be 2 lists of the same length (e.g. burst_starts and burst_ends). That's just one option. Whatever you think is best.

Reply:
Thanks for taking the time to review this!

Yeah, `burst_edges` alternates between cycles before/after bursts. I like your idea of having 2 lists - that would make things clearer. I'll push that change soon.